packages/expo-updates/ios/EXUpdates/UpdatesUtils.swift

```diff
@@ -96,6 +94,26 @@ public final class UpdatesUtils: NSObject {
     return config.runtimeVersion ?? config.sdkVersion ?? "1"
   }
 
+  internal static func embeddedAssetsMap(withConfig config: UpdatesConfig, database: UpdatesDatabase?) -> [String: Any] {
+    var assetFilesMap: [String: Any] = [:]
+    let embeddedManifest: Update? = EmbeddedAppLoader.embeddedManifest(withConfig: config, database: database)
+    let embeddedAssets = embeddedManifest?.assets() ?? []
+
+    // Prepopulate with embedded assets
+    for asset in embeddedAssets {
+      if let assetKey: String = asset.key {
+        if !asset.isLaunchAsset {
+          let absolutePath = path(forBundledAsset: asset)
+          let message = "AppLauncherWithDatabase: embedded asset key = \(asset.key ?? ""), main bundle filename = \(asset.mainBundleFilename ?? ""), path = \(absolutePath ?? "")"
+          NSLog(message)
```

Comment:
should this use a UpdatesLogger?

Reply:
Actually this can be removed. If UpdatesLogger, should probably be debug level.